arlo_server/routes.py

```diff
@@ -437,11 +458,9 @@ def update_jurisdictions_file(election_id=None):
             400,
         )
 
-    bulk_update_jurisdictions(
-        db.session,
-        election,
-        [(row[JURISDICTION_NAME], row[ADMIN_EMAIL]) for row in jurisdictions_csv],
-    )
+    if old_jurisdictions_file:
+        db.session.delete(old_jurisdictions_file)
+    db.session.add(election)
```

Comment:
Is there a reason the `election` needs to be added to the session? I don't quite understand SQLAlchemy yet so I'm curious

Reply:
I don't quite know for sure. I've been conservative about my assumptions on how smart it is and just add anything that changes. In this case it's changing `Election#jurisdictions_file_id` so I'm adding it.